test/helpers/pkcs12.c

```diff
@@ -338,13 +338,13 @@ static STACK_OF(PKCS12_SAFEBAG) *decode_contentinfo(STACK_OF(PKCS7) *safes, int
  * PKCS12 safeBag/attribute builder
  */
 
-static int add_attributes(PKCS12_SAFEBAG *bag, const PKCS12_ATTR *attrs)
+static int add_attributes(PKCS12_SAFEBAG *bag, const PKCS12_ATTR *attr)
 {
     int ret = 0;
     int attr_nid;
-    const PKCS12_ATTR *p_attr = attrs;
+    const PKCS12_ATTR *p_attr = attr;
 
-    if (attrs == NULL)
+    if (attr == NULL)
```

Comment:
You seem to have dropped the changes that you previously had in this file to correctly handle "NID_oracle_jdk_trustedkeyusage". I think those changes were correct (apart from the mem leak)...the default handling here won't quite create the attribute correctly I think.

Reply:
OK I'll re-add that